src/qt/notifymnemonic.cpp

```diff
@@ -0,0 +1,39 @@
+#include "notifymnemonic.h"
+#include "ui_notifymnemonic.h"
+
+#include "guiutil.h"
+
+#include "util.h"
+
+#ifdef ENABLE_WALLET
+#include "walletmodel.h"
+#endif
+
+#include <boost/filesystem.hpp>
+
+#include <QFileDialog>
+#include <QSettings>
+#include <QMessageBox>
+
+NotifyMnemonic::NotifyMnemonic(QWidget *parent) :
+        QDialog(parent),
+        ui(new Ui::NotifyMnemonic)
+{
+    ui->setupUi(this);
+}
+
+NotifyMnemonic::~NotifyMnemonic()
+{
+    delete ui;
+}
+
+void NotifyMnemonic::notify()
+{
+    NotifyMnemonic notify;
+    SecureString mnemonic;
+    SecureString passphrase;
+    pwalletMain->GetHDChain().GetMnemonic(mnemonic, passphrase);
+    notify.ui->mnemonic->setText(mnemonic.c_str());
+    if(notify.exec())
```

Comment:
This call is causing a failure (on Mac at least). 
When creating a new wallet, this function is called. It looks like the `exec` function doesn't exist.
Output:
```
************************
UNKNOWN EXCEPTION       
zcoin in Runaway exception  
```

GUI says this:
`A fatal error occurred. Zcoin can no longer continue safely and will quit.`

Reply:
fixed